packages/call-to-action/index.js

```diff
@@ -38,6 +38,7 @@ function CallToAction(props) {
                   <Button
                     // eslint-disable-next-line react/no-array-index-key
                     key={stableIdx}
+                    className="g-btn"
```

Comment:
Yep, correct, for overrides. One alternative might be to try to tackle all these components at once in a big assembly PR. Swapping in these classNames temporarily felt like a decent low-effort stopgap to avoid a giant PR or drawn-out assembly process.